src/main.rs

```diff
@@ -240,6 +245,30 @@ fn get_languages_from_env() -> Vec<String> {
     )
 }
 
+fn spawn_editor(custom_pages_dir: &Path, file_name: &str) -> anyhow::Result<()> {
+    let _ = create_dir_all(custom_pages_dir);
+    if !custom_pages_dir.is_dir() {
+        return Err(anyhow!("Fail to create dir {:?}", custom_pages_dir));
+    }
```

Comment:
Now that we have `test_recreate_dir`, we can see that doing the following change:
```suggestion
    create_dir_all(custom_pages_dir).context("Failed to create custom pages directory")?;
```
... does _not_ break `test_recreate_dir`, so we can use this after all :)